public/app/features/explore/hooks/useStateSync/index.ts

```diff
@@ -358,7 +358,7 @@ const urlDiff = (
 } => {
   const datasource = !isEqual(currentUrlState?.datasource, oldUrlState?.datasource);
   const queries = !isEqual(currentUrlState?.queries, oldUrlState?.queries);
-  const range = !areRangesEqual(currentUrlState?.range || DEFAULT_RANGE, oldUrlState?.range || DEFAULT_RANGE);
+  const range = !isEqual(currentUrlState?.range || DEFAULT_RANGE, oldUrlState?.range || DEFAULT_RANGE);
```

Comment:
this changes the behavior when the properties in the range objects are not in the same order, are we ok with it?

Reply:
I think lodash ignores the order of properties:

```typescript
import _ from 'lodash'
    
let a = {};
a.from = 1;
a.to = 2;

let b = {}
b.to = 2;
b.from = 1;

console.log("a =", a);
console.log("b =", b);
console.log("_.isEqual(a,b)", _.isEqual(a,b));
```

Console:

```
a =(2) {from: 1, to: 2}
b =(2) {to: 2, from: 1}
_.isEqual(a,b) true
```